<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWidget</class>
 <widget class="QWidget" name="PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,100">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="5" column="1">
          <widget class="QDateTimeEdit" name="maxDateTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="maximumDate">
            <date>
             <year>7930</year>
             <month>12</month>
             <day>31</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1780</year>
             <month>9</month>
             <day>14</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>dd-MM-yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="dateTimePreset">
           <item>
            <property name="text">
             <string>Today</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yesterday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last Month</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateTimeEdit" name="minDateTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="maximumDate">
            <date>
             <year>7930</year>
             <month>12</month>
             <day>31</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1780</year>
             <month>9</month>
             <day>14</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>dd-MM-yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="useDateTimePreset">
           <property name="text">
            <string>Preset</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="4" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="4">
          <widget class="QDoubleSpinBox" name="maxTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QDoubleSpinBox" name="maxHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QCheckBox" name="fitMeasurements">
           <property name="text">
            <string>Fit Measurements</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="minHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="minTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>150.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="selectSensors">
           <property name="text">
            <string>Select Sensors</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/sensor.png</normaloff>:/icons/ui/sensor.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QPushButton" name="exportData">
           <property name="text">
            <string>Export</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/export.png</normaloff>:/icons/ui/export.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QPushButton" name="clearAnnotations">
           <property name="text">
            <string>Clear Annotations</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/remove.png</normaloff>:/icons/ui/remove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QCheckBox" name="useSmoothing">
           <property name="text">
            <string>Smooth Results</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="resultCount">
           <property name="text">
            <string>0 results</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>useDateTimePreset</tabstop>
  <tabstop>dateTimePreset</tabstop>
  <tabstop>minDateTime</tabstop>
  <tabstop>maxDateTime</tabstop>
  <tabstop>fitMeasurements</tabstop>
  <tabstop>minTemperature</tabstop>
  <tabstop>maxTemperature</tabstop>
  <tabstop>minHumidity</tabstop>
  <tabstop>maxHumidity</tabstop>
  <tabstop>useSmoothing</tabstop>
  <tabstop>selectSensors</tabstop>
  <tabstop>exportData</tabstop>
  <tabstop>clearAnnotations</tabstop>
 </tabstops>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimePreset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>minDateTime</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxDateTime</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitMeasurements</sender>
   <signal>toggled(bool)</signal>
   <receiver>minTemperature</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitMeasurements</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTemperature</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitMeasurements</sender>
   <signal>toggled(bool)</signal>
   <receiver>minHumidity</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitMeasurements</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxHumidity</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
